snapcraft/tests/test_common.py

```diff
@@ -112,8 +111,13 @@ class ArchTestCase(testtools.TestCase):
 
     def setUp(self):
         super().setUp()
-        common.host_machine = platform.machine()
-        common.target_machine = common.host_machine
+        # tests will override host_machine, protect it first
+        ph = patch('snapcraft.common.host_machine', new=common.host_machine)
+        ph.start()
+        self.addCleanup(ph.stop)
+        pt = patch('snapcraft.common.target_machine', new=common.host_machine)
+        pt.start()
+        self.addCleanup(pt.stop)
```

Comment:
Should we actually protect these with get/set functions in common and then take care of this issue similar to how we've done the [other stuff in common](https://github.com/ubuntu-core/snapcraft/blob/master/snapcraft/tests/__init__.py#L53)?